### DISCUSSION TITLE
Export skeleton structure to COLLADA file

### DISCUSSION THREAD

**Discussion by mobereng** [libigl/libigl#1774] [Q&A]
Hi, I have been working on skeletal animations using the library, but I was wondering if there is a way to export a model with a skeletal animation.

My current data structure are just this matrices:

```c++
MatrixXd V, C, W;
MatrixXi F, BE;
```

Where `C BE W` contains the skeleton data and `V F` the model, my goal is to generate a single `COLLADA` or `FBX` file with all this information

Thanks in advance !

**Comment by jdumas** [comment]
You should look into Assimp if you want to export to FBX with bones.

**Reply by mobereng** [reply]
Yes, I'm already doing the export using Assimp, so all good there. One last question, is there any function on libigl ot retrive the bones transformation matrices ? as long as I have a simple "graph" like data, I need to build the 4x4 position/rotation matrices for each one of my bones
